audio_handler.py

```
@@ -0,0 +1,315 @@
+                except queue.Empty:
+                    continue
+                except Exception as e:
+                    logger.error(f"Error streaming audio: {e}")
+            else:
+                await asyncio.sleep(0.1)
+                
+    def start_websocket_server(self):
+        """Start WebSocket server for audio streaming"""
+        async def run_server():
+            # Start audio streaming task
+            stream_task = asyncio.create_task(self.stream_audio_data())
```

Ruff (0.12.2)

Local variable `stream_task` is assigned to but never used

Remove assignment to unused variable `stream_task`

(F841)